loadtest/control/simplecontroller/controller.go

```diff
@@ -49,19 +50,26 @@ func (c *SimpleController) Run() {
 			waitAfter: 1000,
 		},
 		{
-			run:       c.createPost,
+			run: func() control.UserStatus {
+				// Do not perform a full reload
+				// on a first run.
+				if i == 0 {
+					return c.reload(false)
+				}
+				return c.reload(true)
```

Comment:
I don't think we need this check. After a login, `reload` should always be called with `false` since we don't need to disconnect/reconnect (we just did it). Each iteration of the actions loop should be the same in `SimpleController` and it should terminate with the user logging out. The only exception would be signing up which is done once. The full reload should be done later on in the loop (somewhere in the middle for this simple controller case but randomly in the future) to simulate a page refresh.

Reply:
Ah sorry, I misunderstood. Yes that makes sense.